eli5/sklearn/explain_prediction.py

```diff
@@ -111,3 +106,13 @@ def _add_intercept(X):
         return sp.hstack([X, intercept]).tocsr()
     else:
         return np.hstack([X, intercept])
+
+
+def _get_X(doc, vec=None, vectorized=False):
```

Comment:
:) I hope I didn't change too many unrelated things! I don't have any more global changes in the pipeline, just adding prediction explanation for regressors.